docs/dev/lsp-extensions.md

```diff
@@ -573,3 +573,14 @@ This request is sent from client to server to open the current project's Cargo.t
 ```
 
 `experimental/openCargoToml` returns a single `Link` to the start of the `[package]` keyword.
+
+## CodeLens resolve request
+
+This request is sent from client to server to resolve previously provided CodeLens.
+
+As an alternative to `any` type in `data` field of `CodeLens`, you may use `CodeLensResolveData`:
+```typescript
+interface CodeLensResolveData {
+    data: (DefinitionParams | TextDocumentPositionParams),
+}
+```
```

Comment:
This is standard LSP command, so it's fine to just update the hash.

https://microsoft.github.io/language-server-protocol/specification#codeLens_resolve

We might want to document all client-side commands we use, but I think that's better done separatly (as I think not only codelens do this)